src/videojs-contrib-hls.js

```diff
@@ -486,6 +491,12 @@ class HlsHandler extends Component {
       this.masterPlaylistController_.audioTracks_.forEach((track) => {
         this.tech_.audioTracks().addTrack(track);
       });
+      // clear current textTracks
+      this.tech_.clearTracks('text');
+      this.masterPlaylistController_.textTracks_.forEach((track) => {
+        this.tech_.textTracks().addTrack_(track);
+      });
+      this.tech_.trigger('loadedsubtitles');
```

Comment:
I don't believe we utilize this (nor can we for native implementations, since it's never fired)